features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Shared/AppsAndKeys/TokenManager.jsx

```diff
@@ -125,6 +129,19 @@ class TokenManager extends React.Component {
         }
     }
 
+    /**
+     * Check if the current user is the owner of the application
+     * @param {*} owner required param
+     */
+    checkOwner(owner) {
+        const username = AuthManager.getUser().name;
+        if (username.includes(owner)) {
+            this.setState({ isUserOwner: true });
```

Comment:
you could simplify this if-else block by

```javascript
this.setState({ isUserOwner: username.includes(owner) });
```

Reply:
resolved